tests/vmi_multus_test.go

```diff
@@ -458,9 +458,6 @@ var _ = Describe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with invalid MAC addres", func() {
-			BeforeEach(func() {
```

Comment:
Why did you remove the `BeforeEach`?

Reply:
I've update the `BeforeTestCleanup` to remove the net-attach-defs, as per @rmohr 's comment in https://github.com/kubevirt/kubevirt/pull/2935#discussion_r505471830. 

That entry was causing the NADs to be removed (since they are only created in the `BeforeAll` section).

If you take a look around, you'll furthermore see that test is the only one with this `BeforeEach` block.